index.html

```diff
@@ -13,16 +13,16 @@
             <div class="flexend wide">
                 <nav><a class="menubar" href="#" onclick="showMenu()"><img class="menu-button" src="images/menu.png" alt="menu"></a>
                     <ul class="hormenu">
-                        <li class="link"><a href="#">Hello</a></li>
-                        <li class="link"><a href="#">Portfolio</a></li>
-                        <li class="link"><a href="#">About</a></li>
-                        <li class="link"><a href="#">Contact</a></li>
+                        <li class="link"><a href="#Hello">Hello</a></li>
+                        <li class="link"><a href="#Projects">Projects</a></li>
+                        <li class="link"><a href="#About">About</a></li>
+                        <li class="link"><a href="#Contact">Contact</a></li>
                     </ul>
                 </nav>
             </div>
         </header>
     </div>  
-        <section class="padd">
+        <section class="padd" id="Hello">
             <div class="gridbox">
                 <div class="grid1">
                 <h1>Charlie<br>Giorgania</h1>
```

Comment:
- [ ] Kindly here your headline overlaps the image, kindly fix it to match the requirements with the `Resize Text` point.

![image](https://user-images.githubusercontent.com/35535130/161755420-450e7c08-33b0-44b3-886f-19b20a857906.png)